index.js

```diff
@@ -24,8 +24,14 @@ io.on("connection", (socket) => {
   socket.on("send_data", (data) => {
     socket.broadcast.emit("receive_data", data)
   })
+
+  socket.on("send_address", (data => {
```

Comment:
so the socket turns on whenever you send_address and it will emit the recieve address?